Source/WebCore/rendering/PathOperation.h

```diff
@@ -103,6 +102,7 @@ class ShapePathOperation final : public PathOperation {
 
     void setReferenceBox(CSSBoxType referenceBox) { m_referenceBox = referenceBox; }
     CSSBoxType referenceBox() const { return m_referenceBox; }
+    const std::optional<Path> getPath(FloatRect reference = FloatRect(), FloatPoint anchor = FloatPoint(), OffsetRotation rotation = OffsetRotation(false, 0)) const override { UNUSED_PARAM(anchor); UNUSED_PARAM(rotation); return pathForReferenceRect(reference); }
```

Comment:
`final`. You should not need the default values here, and you can just not name the parameters in this override.

Reply:
Fixed.